examples/tts/speechllm/conf/megatron_t5_speechllm_inference.yaml

```diff
@@ -100,6 +105,10 @@ model:
     speech_offset: 30000
     train_task: asr
```

Comment:
Should we update the task on both inference yamls?

Reply:
Yes, my bad. Let me update this.